docs/src/development/transfer-dedicated.md

```diff
@@ -51,20 +51,28 @@ Which should give a JSON output containing something like this:
 ]
 ```
 
-The part you want is the user, password, and "path", which means the DB name.
-Ignore the rest.
+The values needed are:
```

Comment:
:hammer_and_wrench: I'd switch this to active/direct: `The values you need` or better: `You need the following values`